operations/OperationsManager.go

```diff
@@ -60,6 +62,15 @@ func OperationsInit() (io.Writer, logging.Tracer) {
 		if len(tracers) > 0 {
 			tracer = logging.NewMultiplexedTracer(tracers...)
 		}
+
+		tlsCert, tlsCertErr := identity.GetCert()
+		if tlsCertErr == nil {
+			x509Cert, _ := x509.ParseCertificate(tlsCert.Certificate[0])
+			serverId, err := identity.IDFromCert(x509Cert)
+			if err == nil {
```

Comment:
Isn't better to move this snippet to the quicsec/identity?  We should avoid touching the certificate outside Indentity or AuthManager modules. IDFromCert can call GetCert() and ParseCertifiate() internally.

Reply:
Move the logic to identity.GetCurrentIdentity(), tks for the suggestion